README.md

```diff
@@ -97,7 +97,7 @@ After you're finished please remove all the comments and instructions!
 
```

Comment:
- [ ] Please make sure the video contains a highlight about a piece of the code the you finds interesting or is proud of making.